packages/l10n/scripts/generate-l10n-data.js

```diff
@@ -294,6 +295,79 @@ const mapDiffToWarnings = (oldData, newData) => {
     .filter(Boolean);
 };
 
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const updateTimeZoneTranslations = async (key) => {
+  const targetTranslationsFolder = path.join(
+    __dirname,
+    '..',
+    DATA_DIR[key].path
+  );
+  const targetTranslationsFile = path.join(
+    targetTranslationsFolder,
+    'core.json'
+  );
+  /**
+   * This is a list of all timezones in the output of `moment.tz.names()`
+   * that we want to exclude from the final compiled time zone data.
+   *
+   * The `extractTimeZoneDataForLocale` function in `scripts/generate-l10n-data`
+   * consumes this list of excluded timezones, and an object of locale-specific
+   * long-form translations.  If a timezone is present in the output of
+   * `moment.tz.names()`, but either is not in the excluded list or does not
+   * have a translation, that timezone is considered a new timezone.
+   *
+   * If a new timezone is encountered, the user running the CLI command
+   * will be prompted via the shell whether they would like to accept
+   * the new timezone, or add that timezone to this excluded list.
+   */
+  const targetExcludedTimeZonesFile = path.join(
+    targetTranslationsFolder,
+    'excluded_time_zones.json'
+  );
```

Comment:
Putting it in `scripts` works for me.  

The only reason I have it as a json file is because there is a potential that we might be writing values to the file programmatically via the cli, but I think you're right that `.js` makes more sense